src/Modal/Modal-styled.js

```diff
@@ -39,34 +46,123 @@ const StyledModalOverlay = {
   opacity: 0
 };
 
-const StyledModalDialog = {
-  position: 'relative',
-  top: 'auto',
-  left: 'auto',
-  right: 'auto',
-  bottom: 'auto',
-  maxWidth: '80vw',
-  maxHeight: '80vh',
-  overflow: 'auto',
-  outline: 'none',
-  boxSizing: 'border-box',
-  zIndex: '102',
-  background: theme.palette.white,
-  padding: theme.baseline,
-  textAlign: 'initial',
-  overflowY: 'auto',
-  display: 'inline-block',
-  verticalAlign: 'middle',
-  borderRadius: theme.borderRadius,
-  border: 'none',
-  transition: `margin-top 300ms cubic-bezier(0.4, 0.0, 0.2, 1)`,
-  marginTop: '30px'
+const StyledModalDialog = color => {
```

Comment:
I think this ought to be refactored to emphasize that it is returning options for react-modal's `style` prop instead of naming it like a styled-component.

Reply:
Good note, I'll change the name and leave a code comment.